ui/app/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js

```diff
@@ -140,6 +141,8 @@ export default class ImportWithSeedPhrase extends PureComponent {
       })
 
       setSeedPhraseBackedUp(true).then(() => {
+        const address = vault.keyrings[0].accounts[0]
+        initializeThreeBox(address)
```

Comment:
Maybe this would be better done in `metamask-controller`? I found it a bit hard to keep track of where 3Box was being initialized in each of the three scenarios: unlocking, creating a new account, and restoring an account. The first two cases are in the controller, only the last is in the UI.